src/gwt/src/org/rstudio/studio/client/application/Application.java

```diff
@@ -890,6 +899,13 @@ public void execute(String theme)
             commands_.newSession().remove();
       }
       
+      // show support link only in RStudio Pro
+      if (pEdition_.get() != null)
```

Comment:
When might this be null? I'm not checking for this elsewhere on injected objects.

Reply:
That's a good question. It's exceedingly unlikely, but not wholly impossible, for a failure to occur when gin tries to create the instance. So you're almost certainly fine, I'm just being extra paranoid here because this is in `initWorkbench` so a `NullPointerException` could have disastrous consequences.